lomond/session.py

```diff
@@ -13,10 +13,14 @@
 import socket
 import ssl
 import threading
-import time
 
 from six.moves.urllib.parse import urlparse
 
+try:
+    from monotonic import monotonic as monotonic_time
+except:
+    from time import time as monotonic_time
```

Comment:
If its imported this way, it's not strictly `monotonic_time`. Suggest calling it `get_time`